src/Nancy/RequestHeaders.cs

```diff
@@ -112,27 +112,27 @@ public string Connection
         /// <value>The length of the contents if it is available; otherwise 0.</value>
         public long ContentLength
         {
-            get { return this.GetValue("Content-Length", x => Convert.ToInt64(x.First())); }
+            get { return this.GetValue("Content-Length", x => Convert.ToInt64(x.First()), 0); }

```

Comment:
Quite possibly, but that's a change that would be unrelated to this pull-request